onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -344,6 +344,7 @@ void InitializeSession(InferenceSession* sess, const std::vector<std::string>& p
 }
 
 void addGlobalMethods(py::module& m) {
+  m.def("get_default_session_options", &GetDefaultCPUSessionOptions, "Return a default session_options instance.");
```

Comment:
Add provision to fetch a default Session Options instance. This allows us to get rid of the redundant py::init() for the InferenceSession class below...

Reply:
thanks. makes sense.